StackExchange/Stacks
# docs(figma): obfuscate figma links under okta auth

[STACKS-608](https://stackoverflow.atlassian.net/browse/STACKS-608)

This PR obfuscate figma links under okta auth.

See also:
- https://stackexchange.slack.com/archives/C27RWNQN9/p1713382302870629
- https://stackexchange.slack.com/archives/C27RWNQN9/p1713379234110439

### To test

Checkout pages like [Color fundamentals](https://deploy-preview-1712--stacks.netlify.app/product/foundation/color-fundamentals/) or [Box Shadow](https://deploy-preview-1712--stacks.netlify.app/product/base/box-shadow/) and notice that the figma links are now forcing you to go through okta auth before being revealed.

[STACKS-608]: https://stackoverflow.atlassian.net/browse/STACKS-608?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
